### PR TITLE
add query device actions

### PR DESCRIPTION
**Pull request by dougperez**

### CODE REVIEW

File: lib/conduit/cogotech/actions/base.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    class << self
+      def http_method(http_method = nil)
+        @http_method = http_method unless http_method.nil?
+        @http_method
+      end
+    end
+
+    def remote_url
+      "http://#{production_url}"
```
Comment (pedromartinez) [inline lib/conduit/cogotech/actions/base.rb]: Should this be hardcoded to be http?

Comment (dougperez) [inline lib/conduit/cogotech/actions/base.rb]: Prob not, this is what was in the api example. We can change it to be configured.

File: lib/conduit/cogotech/actions/base.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    def_delegators :'self.class', :http_method
+
+    class << self
+      def http_method(http_method = nil)
+        @http_method = http_method unless http_method.nil?
+        @http_method
+      end
+    end
+
+    def remote_url
```
Comment (pedromartinez) [inline lib/conduit/cogotech/actions/base.rb]: Isn't it possible to set this up in the following way?

https://github.com/conduit/conduit/blob/9c60111f60ae987a8990dec8c0d97babd938f575/spec/support/my_driver/actions/foo.rb

File: lib/conduit/cogotech/parsers/query_device_info.rb [hunk 3]
```diff
@@ -0,0 +1,55 @@
+require 'conduit/cogotech/parsers/base'
+
+module Conduit::Driver::Cogotech
+  class QueryDeviceInfo::Parser < Parser::Base
+    attribute :mdn
+    attribute :description
+    attribute :customer_id do
```
Comment (pedromartinez) [inline lib/conduit/cogotech/parsers/query_device_info.rb]: You can define the default behavior of the attribute method to look for camelcase objects.

see:
https://github.com/conduit/conduit-sprint/blob/master/lib/conduit/sprint/parsers/base.rb#L10

Comment (dougperez) [inline lib/conduit/cogotech/parsers/query_device_info.rb]: Thx, that worked.

**Pull request merged**